examples/resources/gworkspace-connector-inline.yaml

```diff
@@ -29,7 +29,7 @@ spec:
      "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/teleport-service-account%40access-304316.iam.gserviceaccount.com"
     }
   issuer_url: https://accounts.google.com
-  redirect_url: https://<cluster-url>:3080/v1/webapi/oidc/callback
+  redirect_url: https://<cluster-url>:443/v1/webapi/oidc/callback
```

Comment:
Should we even put 443 here? In most cases I would not expect the port to be included in the URL here.